hack/generate-etcd-secret.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+cat <<-EOF
+apiVersion: v1
+data:
+  etcd-client-ca.crt: "$(ssh -i ~/go/src/github.com/openshift/release/cluster/test-deploy/gcp-dev/ssh-privatekey cloud-user@35.237.160.60 sudo -E cat /etc/origin/master/master.etcd-ca.crt | base64 --wrap=0)"
```

Comment:
How about making the IP address an argument to the script?

Reply:
Definitely! Good catch.